docs/v7_migration.rst

```diff
@@ -174,6 +174,21 @@ been functional since around October 2022. It was deprecated in ``v6`` and has b
 completely removed in ``v7``.
 
 
+Remaining cameCase -> snake_case Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following arguments have been renamed across the library from camelCase to
+snake_case in all methods where they are passed in as a kwarg.
+
+- ``fromBlock`` -> ``from_block``
+- ``toBlock`` -> ``to_block``
+- ``blockHash`` -> ``block_hash``
+
+Note that if a dictionary is used instead, say to a call such as `eth_getLogs`, the
+keys in the dictionary should be camelCase. This is because the dictionary is passed
+directly to the JSON-RPC request, where the keys are expected to be in camelCase.
```

Comment:
What do you think about adding a formatter for the case of `getLogs` and friends? Basically it would look at the TxDict and flip any snake_case to camelCase to be passed on to the node. Then on the way back out it would do the same.

Reply:
I'd be good with that if the user experience would benefit from it. The way it is now, users are aware the transactions need to have camelCase arguments as well i.e. `{"maxFeePerGas": 10**9, ...}`. I think this is a similar case to that but I do think it's a touch more confusing because of the way the args are passed in. I could go either way.